components/single_image_view/single_image_view.tsx

```diff
@@ -185,7 +184,7 @@ export default class SingleImageView extends React.PureComponent<Props, State> {
                     show={this.state.showPreviewModal}
                     onModalDismissed={this.showPreviewModal}
                     fileInfos={[fileInfo]}
-                    post={this.props.post}
+                    postId={this.props.postId}
```

Comment:
This was changed here in the PR https://github.com/mattermost/mattermost-webapp/pull/6774 most likely because the `ViewImageModal` expects post and we are passing postId here. 

The connected component takes care of getting the post from postId but ts complains hence adding the prop below